README.md

```diff
@@ -7,36 +7,28 @@
       ""     ""    ""                                                          ""    ""     ""
 ```
 
-# cromshell
- A CLI for submitting workflows to a cromwell server and monitoring / querying their results.
+# Cromshell
+[![GitHub version](https://badge.fury.io/gh/broadinstitute%2Fcromshell.svg)](https://badge.fury.io/gh/broadinstitute%2Fcromshell)
```

Comment:
Replaced the "Current version:..." wording with a version badge. There are other badges that can be added later like coverage

[![codecov](https://codecov.io/gh/broadinstitute/cromshell/branch/develop/graph/badge.svg)](https://codecov.io/gh/broadinstitute/cromshell)